wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -248,7 +248,7 @@ static function enqueue_scripts() {
 			)
 		);
 
-		if( $current_screen->id === 'upload' || $current_screen->id === 'post' ||
+		if( $current_screen->id === 'upload' || $pagenow === 'post.php' ||
```

Comment:
Seems like that would probably work too...gotta check both `post.php` and `post-new.php` variants